ui/src/app/pages/admin/user/admin-user-egress-bypass.tsx

```diff
@@ -16,41 +16,41 @@ import { TooltipTrigger } from 'app/components/popups';
 import { userAdminApi } from 'app/services/swagger-fetch-clients';
 import colors from 'app/styles/colors';
 import { displayDate, isDateValid, maybeToSingleDate } from 'app/utils/dates';
-const MIN_BYPASS_DESCRIPTION = 10;
-const MAX_BYPASS_DESCRIPTION = 4000;
 
-interface Props {
-  userId: number;
+export const MIN_BYPASS_DESCRIPTION = 10;
```

Comment:
changes here are also not strictly needed after a switch from a different approach, but they're a bit of cleanup